docs/Configuration.md

```diff
@@ -43,8 +43,7 @@ exporters:
 
 extensions:
   sumologic:
-    access_id: <my_access_id>
-    access_key: <my_access_key>
+    install_token: <token>
```

Comment:
These changes in docs will be problematic **until** we deploy installation token support on prods but 🤷